lib/busbar_cli/services/logs.rb

```diff
@@ -3,9 +3,11 @@ class Logs
     def self.call(container_id:, environment_name:, since:)
       Services::Kube.setup
 
+      container_name = container_id.split('-')[0..-3].join('-')
+
       Kernel.exec(
         "#{KUBECTL} --context=#{Services::Kube.current_profile} " \
-        "logs -f --since=#{since} #{container_id} -n #{environment_name}"
+        "logs -f --since=#{since} #{container_id} #{container_name} -n #{environment_name}"
```

Comment:
Does this fix that bug where busbar would sometimes return logs for the wrong app?